From 73768b8e327066f691188a5f6619d04c4fb56dda Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Wed, 31 May 2017 18:55:09 +0200
Subject: [PATCH] network: Fix initial visibility of summary item

This was fixed when upstreaming the patch, but never made it back
into the RHEL version ...
---
 js/ui/status/network.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/js/ui/status/network.js b/js/ui/status/network.js
index 842f6be6a..e42ebabd5 100644
--- a/js/ui/status/network.js
+++ b/js/ui/status/network.js
@@ -1607,6 +1607,7 @@ const DeviceCategory = new Lang.Class({
 
         this._summaryItem.menu.addSettingsAction(_('Network Settings'),
                                                  'gnome-network-panel.desktop');
+        this._summaryItem.actor.hide();
 
     },
 
-- 
2.13.0

